src/main/scala/intellij/haskell/util/HaskellProjectUtil.scala

```diff
@@ -78,7 +79,12 @@ object HaskellProjectUtil {
   }
 
   def getModuleDir(module: Module): File = {
-    new File(ModuleUtilCore.getModuleDirPath(module))
+    val path = ModuleUtilCore.getModuleDirPath(module)
+    new File(
+      if (path.endsWith(".idea"))
```

Comment:
```suggestion
      if (path == ".idea")
```

Reply:
I think using `==` is more to-the-point :smile:

Reply:
I've run some tests and that will fail for the issue I am adressing

Reply:
the path here is "<User/....../workspace/..../.idea>" not the name of the directory

Reply:
Ok, I see